### ISSUE TITLE
Create Selfkey ID Profile when editing attributes on KYC checklist

### ISSUE THREAD

**Issue by rodrigopavezi** [SelfKeyFoundation/Identity-Wallet#826]

**Comment by rodrigopavezi** [comment]
@oanamangiurea @andre should we route the user to selfkey id creation if she/he hasn't yet? The thing is that KYC-API needs the user email to create a user. So I think user should not start any incorporation before creating a selfkey id and adding an email. Thoughts?

**Comment by andregoncalves** [comment]
I agree.
However, it shouldn't just a simple routing, there should be some explanation somewhere about why he needs to do so.

**Comment by sk91** [comment]
@rodrigopavezi @andregoncalves will create a popup that will appear on incorporation start if no selfkey id.

It will explain that before incorporating they should create their selfkey id, and will have a button to redirect to selfkeyid

**Comment by rodrigopavezi** [comment]
@andregoncalves @oanamangiurea @sk91 I think we should also add a field for entering email to the selfkey id creation form and make it required. Otherwise KYC will create a user with empty email. Cheers

**Comment by sk91** [comment]
@rodrigopavezi I think if they require email in the template, kyc checklist will not allow to incorporate

**Comment by sk91** [comment]
but adding email is a good idea

**Comment by rodrigopavezi** [comment]
@sk91 the challenge request I believe also requires the email.

**Comment by sk91** [comment]
if nothing changed since I first developed the rp client --- than no @rodrigopavezi

**Comment by andregoncalves** [comment]
Should we ping Oana for the text copy?

**Comment by sk91** [comment]
@andregoncalves yes, sure @oanamangiurea ^^.

I will begin with my own text and update once Oana has time for it

**Comment by sk91** [comment]
<img width="1078" alt="screen shot 2019-03-08 at 09 13 10" src="https://user-images.githubusercontent.com/1021965/54013713-dc5ac180-4182-11e9-9a7b-e1024e5c4dd8.png">
<img width="1395" alt="screen shot 2019-03-08 at 09 10 19" src="https://user-images.githubusercontent.com/1021965/54013714-dc5ac180-4182-11e9-8523-400c99478e13.png">

FOR QA: please ignore icon color. It is a global wallet issue and is tracked here: https://github.com/SelfKeyFoundation/Identity-Wallet/issues/879

**Comment by oanamangiurea** [comment]
@sk91 if not too late, from your screenshot above, I see a vertical space issue between last name and email (missing space) https://gyazo.com/068592ad4bb4e91bb0cb24ae21c8decd any chance to fix it as well? I can also open a new issue for it

**Comment by sk91** [comment]
@oanamangiurea will fix, no . problem

**Comment by sk91** [comment]
<img width="773" alt="screen shot 2019-03-08 at 13 49 24" src="https://user-images.githubusercontent.com/1021965/54027152-1d65cc80-41a9-11e9-964f-837c1c99aec1.png">

**Comment by oanamangiurea** [comment]
Perfect now, thank you!
